common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -493,6 +494,9 @@ def _student_or_public_view(self, context, prereq_met, prereq_meta_info, banner_
 
         fragment = Fragment()
         params = self._get_render_metadata(context, display_items, prereq_met, prereq_meta_info, banner_text, view, fragment)
+        if settings.FEATURES.get('SHOW_PROGRESS_BAR', False):
+            parent_block_id = self.get_parent().scope_ids.usage_id.block_id
+            params['chapter_completion_aggregator_url'] = settings.COMPLETION_AGGREGATOR_URL + '/' + six.text_type(self.course_id) + '/' + parent_block_id + '/'
```

Comment:
@pkulkark  how about using the following instead? That's a bit more clear I think:

```suggestion
            params['chapter_completion_aggregator_url'] = '/'.join([settings.COMPLETION_AGGREGATOR_URL, str(self.course_id), parent_block_id])
```

Please note that the trailing slash is not appended in this case, though I guess that's not needed (but you know that better). If that's needed it is better to add a simple `+ '/'` trailing slash.